css/selectors/matches.json

```diff
@@ -70,19 +105,35 @@
               },
               {
                 "version_added": "5",
-                "alternative_name": ":-webkit-any"
+                "alternative_name": ":-webkit-any",
+                "notes": [
+                  "It does not support combinators."
+                ]
               }
             ],
             "samsunginternet_android": {
               "version_added": false
             },
             "webview_android": [
               {
-                "version_added": "66"
+                "version_added": "66",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#enable-experimental-web-platform-features",
+                    "value_to_set": "enabled"
+                  }
```

Comment:
I don’t think that WebView supports changing preferences.

needinfo?(@jpmedley)